BundleTransport/app/src/main/java/net/discdd/bundletransport/utils/LocalReportSender.java

```diff
@@ -0,0 +1,63 @@
+package net.discdd.bundletransport.utils;
+
+import android.content.Context;
+
+import com.google.auto.service.AutoService;
+import org.acra.config.CoreConfiguration;
+import org.acra.data.CrashReportData;
+import org.acra.sender.ReportSender;
+import org.acra.sender.ReportSenderException;
+import org.acra.sender.ReportSenderFactory;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Path;
+
+public class LocalReportSender implements ReportSender {
+    CoreConfiguration config;
+
+    public LocalReportSender(CoreConfiguration coreConfiguration) {
+        config = coreConfiguration;
+    }
+
+    @Override
+    public void send(Context context, CrashReportData errorContent) throws ReportSenderException {
+        // Iterate over the CrashReportData instance and do whatever
+        // you need with each pair of ReportField key / String value
+
+        //declare root dir and append target dir (toServer)
+        Path rootDir = context.getApplicationContext().getExternalFilesDir(null).toPath();
+        Path destDir = rootDir.resolve("BundleTransmission/server");
+        if (destDir.toFile().exists()) {
+            File logFile = new File(String.valueOf(destDir), "crash_report.txt");
```

Comment:
this line is repeated in the else, you should pull it out and put it below the if

Reply:
oh actually it is already there. you can just remove this line and the other in the else clause